frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1946,6 +2007,19 @@ mod tests {
 			roll_to(30);
 			assert!(MultiPhase::current_phase().is_unsigned_open_at(20));
 
+			// tries election before the minimum election blocks, fails election, do not try
```

Comment:
Suggestion: 

For each test, the default value of `MinBlocksBeforeEmergency` should be `SignedPhase + unsigedPhase` (not hardocded, you can actually use an `expr` in `parameter_types`).

This guarantees that no existing test should break, expcet if the test is intentionally calling into elect early, which we have tests for.

This leaves most existing tests untouched, and allowes you to only add new tests.

Reply:
For example, this test is about "what if this pallet is configured with signed phase 0", but now we're mixing it with throttling logic.